keras/backend/jax/random.py

```diff
@@ -83,7 +83,7 @@ def dropout(inputs, rate, noise_shape=None, seed=None):
 
 def shuffle(x, axis=0, seed=None):
     seed = jax_draw_seed(seed)
-    return jax.random.shuffle(seed, x, axis)
+    return jax.random.permutation(seed, x, axis, independent=True)
```

Comment:
Can you provide some explanation about `independent=True`? The docs weren't super helpful unfortunately https://jax.readthedocs.io/en/latest/_autosummary/jax.random.permutation.html

Reply:
Oh I see it's what is recommend in the deprecation message.